src/tools/dev/scripts/bv_support/bv_main.sh

```diff
@@ -402,10 +369,14 @@ function initialize_build_visit()
             export MACOSX_DEPLOYMENT_TARGET=11.0
             export C_COMPILER=${C_COMPILER:-"clang"}
             export CXX_COMPILER=${CXX_COMPILER:-"clang++"}
-        else
-            export MACOSX_DEPLOYMENT_TARGET=10.14
+        elif [[ ${VER_MAJOR} == 21 ]] ; then
```

Comment:
we do need the `MACOSX_DEPLOYMENT_TARGET`, but for all cases on mac we will use `clang` and `clang++`, so we can simplify this logic further